src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -102,6 +102,8 @@ internal PackageSourceMoniker SelectedSource
 
         private readonly Guid _sessionGuid = Guid.NewGuid();
         private readonly Stopwatch _sinceLastRefresh;
+        private bool _installedTabIsLoaded;
```

Comment:
the term "IsLoaded" can be misinterpreted here.
Do you mean, "HasBeenLoaded"...

It isn't necessarily currently loaded (or display)...but it has been loaded...

Reply:
or change it to:
_installedDataIsLoaded
_updatesDataIsLoaded

---
In reply to: [450524712](https://github.com/NuGet/NuGet.Client/pull/3444#discussion_r450524712) [](ancestors = 450524712)

Reply:
Being loaded != being visible. HasBeen to me sounds like... it was at some point, but not necessarily right now.

Reply:
yes, i'd recommend my 2nd comment in this thread over hasbeen...    DataIsLoaded, not TabLoaded.
There is a Loaded event in WPF on FrameworkElement. It means it has been parsed and created (ish).

Here we are talking about the Data being loaded earlier, please reuse it, -- not that the UI elements have been loaded, i think.

---
In reply to: [450529298](https://github.com/NuGet/NuGet.Client/pull/3444#discussion_r450529298) [](ancestors = 450529298)